src/renderer.zig

```diff
@@ -166,6 +168,29 @@ pub fn render(playerPosition: Vec3d) void {
 	}
 }
 
+pub fn crosshairDirection(rotationMatrix: Mat4f, fovY: f32, width: u31, height: u31) Vec3f {
+	// stolen code from Frustum.init
+	const invRotationMatrix = rotationMatrix.transpose();
+	const cameraDir = vec.xyz(invRotationMatrix.mulVec(Vec4f{0, 0, 1, 1}));
+	const cameraUp = vec.xyz(invRotationMatrix.mulVec(Vec4f{0, 1, 0, 1}));
+	const cameraRight = vec.xyz(invRotationMatrix.mulVec(Vec4f{1, 0, 0, 1}));
+
+	const screenSize = Vec2f{@floatFromInt(width), @floatFromInt(height)};
+	const screenCoord = crosshair.window.pos + crosshair.window.contentSize*Vec2f{0.5, 0.5};
+
+	const halfVSide = std.math.tan(std.math.degreesToRadians(f32, fovY)*0.5);
```

Comment:
I guess you tested with an older version of zig. The current version of `degreesToRadians` only takes one argument.

Reply:
Fortunately the happy path is to just use `run_linux.sh`. Most likely I did that, but it stopped at an earlier error. Speaking of which, I think it makes sense to have Cubyz come to compiling terms with MacOS, if only to open up developer support.